setup_environment.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
```

Comment:
This script is well written and can be also used to install enviroment on "unclean" system by installing only missing stuff.
In this way development can be done on machines shared with other users (i.e. my wife won't complained that I broken her Java enviroment ;) ).